clients/java/src/test/java/marquez/client/MarquezWriteOnlyClientTest.java

```diff
@@ -89,7 +88,7 @@ public void testCreateRunWithId() {
     RunMeta runMeta = RunMeta.builder().id(id).build();
     client.createRun("namespaceName", "jobName", runMeta);
     verify(backend, times(1))
-        .post("/api/v1/namespaces/namespaceName/jobs/jobName/runs", runMeta.toJson());
```

Comment:
what do you think about handling this case also in method?
`private URL url(String path) {`

Reply:
Any idea of something better than 
```java
      if (baseUrl.toString().endsWith("/") && path.startsWith("/")) {
        path = path.substring(1);
      } else if (!baseUrl.toString().endsWith("/") && !path.startsWith("/")) {
        path = "/" + path;
      }
```